test/jit_hooks/model.py

```diff
@@ -0,0 +1,515 @@
+import argparse
```

Comment:
The classes and tests in this file look identical to those in the unit tests from the PR below this one. If that is the case, I think we should explore ways to put them in some common location and use them in the Python unit tests. The only difference appears to be that there are no error tests here (which makes sense) and these modules are exported instead of being passed to `checkModule`.

Reply:
I moved the modules and hooks to a new shared file 👍